src/content/docs/en/reference/api-reference.mdx

```diff
@@ -2054,3 +2083,75 @@ This component provides a way to inspect values on the client-side, without any
 
 
 [canonical]: https://en.wikipedia.org/wiki/Canonical_link_element
+
+## Actions (astro:actions)
+
```

Comment:
I think it's better to use the stable version, so I used the version mentioned in this PR. It will need to be changed if the release is delayed!

```suggestion

<p>

<Since v="4.15.0" />
</p>

```